apps/tenant-management-webapp/src/app/pages/admin/services/access/access.tsx

```diff
@@ -30,12 +30,11 @@ export default function (): JSX.Element {
       </Main>
 
       <Aside>
-        <SupportLinks />
-
         <Gapadjustment>Helpful links</Gapadjustment>
```

Comment:
Could change it to GapAdjustment and HyperlinkColor? thanks!

Reply:
updated